### PR TITLE
Add support for Ubuntu repositories

### PR DESCRIPTION
**Pull request by markgoddard**
Adds support for using Ubuntu as a host and container OS.

* Sync new Ubuntu Focal repositories in Ark to the local Pulp server
* Configure hosts to use the APT repositories on the local Pulp server
* Build Kolla container images, using Ubuntu repositories in the local Pulp server

### CODE REVIEW

File: etc/kayobe/apt.yml [hunk 1]
```diff
@@ -11,6 +11,58 @@
...
+# * filename: name of a file in /etc/apt/apt.conf.d/ in which to write the
+#   configuration
+# Default is an empty list.
+apt_config:
+  # NOTE: Currently the Pulp verbatim publisher does not sync translation
+  # files, which results in apt update failing. Disable translations until this
+  # is resolved.
+  - content: |
+      Acquire::Languages "none";
+    filename: 99no-languages
```
Comment (priteau) [inline etc/kayobe/apt.yml:26]: Should the second line be aligned with the first?

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -220,6 +220,10 @@ kolla_build_blocks:
     {% endfor %}
   grafana_plugins_install: |
     RUN grafana-cli plugins install vonage-status-panel
+  ironic_conductor_header: |
+    # Fix dependency resolution taking forever on Ubuntu.
+    # Remove when https://review.opendev.org/c/openstack/ironic/+/849491 lands.
```
Comment (priteau) [inline etc/kayobe/kolla.yml]: This is being merged.

File: etc/kayobe/apt.yml (hunk 1 above)
Comment (markgoddard) [reply etc/kayobe/apt.yml:26]: No - this is a dict with keys content and filename

File: etc/kayobe/kolla.yml (hunk 2 above)
Comment (markgoddard) [reply etc/kayobe/kolla.yml]: Removed from the commit history.

File: etc/kayobe/apt.yml (hunk 1 above)
Comment (priteau) [reply etc/kayobe/apt.yml:26]: Makes sense!

**Review by priteau — APPROVED** [review]

**Pull request merged**